third_party/rust/repos.bzl

```diff
@@ -95,11 +95,13 @@ def rust_repos(rules_rust = None, safe_ftdi = None, serde_annotate = None):
     http_archive_or_local(
         name = "rules_rust",
         local = rules_rust,
-        sha256 = "1a110ae8bfa34091e2a62e4c0325f8231f934b2956f3a585d4cafba5c81c2676",
-        strip_prefix = "rules_rust-rebase-20231115_01",
-        url = "https://github.com/lowRISC/rules_rust/archive/refs/tags/rebase-20231115_01.tar.gz",
+        sha256 = "e85891dd6bb24e877a9622c09f028b9e11c0f4df69db4735194eb1d8dc7ae917",
+        strip_prefix = "rules_rust-rebase-20230822_02",
+        url = "https://github.com/lowRISC/rules_rust/archive/refs/tags/rebase-20230822_02.tar.gz",
```

Comment:
The version bump is there to fix an issue found in OT cert gen, please don't downgrade.

Reply:
Sorry this is a rebase snafu.
I had to change this temporarily on my machine due to https://github.com/lowRISC/opentitan/issues/20577.
I removed it from this PR.